scripts/operator/raise_pr_to_community_operators_from_our_fork.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+import os
+from git import Repo
+from github import Github
+import shutil
+from get_last_published_operator_version import getLastOperatorVersion
+import pr_resources
+
+
+def get_latest_operator_branch():
+    repo = Repo.clone_from(
+        'https://github.com/snyk/community-operators.git', './community-operators-repo')
+
+    operator_branches = []
+
+    for refs in repo.refs:
+        if 'snyk-operator' in refs.name:
+            operator_branches.append(refs.name)
+
+    return operator_branches[-1].split('origin/')[-1]
+
+
+def create_pull_request(new_operator_version, new_release_branch_name):
+    github = Github('23ba6faa9635ff064f7bf6a6c36a0a33dff4f376')
```

Comment:
Should we put this id in an env var (and in 1Password to make it easier to get it for development)?

Reply:
Yes, I'll add an env var